db/dev_seeds/sdg.rb

```diff
@@ -1,3 +1,32 @@
 section "Creating Sustainable Development Goals" do
   load Rails.root.join("db", "sdg.rb")
+
+  SDG::Target.sample(30).each do |target|
+    title = "Title for default locale"
+    description = "Description for default locale"
+    rand(2..3).times do |n|
+      local_target = SDG::LocalTarget.create!(code: "#{target.code}.#{n}",
```

Comment:
I see targets are created with the code `0`, so I guess it should be `n + 1` :thinking:.